src/main/java/gov/nasa/pds/validate/commandline/options/Flag.java

```diff
@@ -69,6 +69,7 @@ public enum Flag {
       "Specify file extension for the labels files. Default: xml. NOTE: Support for intermingled bundles where collections have differing label file extensions is not yet supported."),
 
   EVERY_N(null, "everyN", "value", int.class, "Process every N files with the default being every file."),
+  CONTEXT_MISMATCH_AS_WARNING(null, "context-mismatch-as-warning", "When true, context mismatches will reported as warnings otherwise as info"),

```

Comment:
@al-niessner per my comment on the ticket, this should be the default functionality, with an ability to disable the warning messages if you do not want them to appear.

Reply:
done

First failure shows they are warnings. Then fixed tests to turn them back into infos. Now tests pass.